gfx.h

```diff
@@ -1424,8 +1424,9 @@ class GfxInternal
         swap_chain_desc.Format           = DXGI_FORMAT_R8G8B8A8_UNORM;
         swap_chain_desc.BufferCount      = max_frames_in_flight_;
         swap_chain_desc.BufferUsage      = DXGI_USAGE_RENDER_TARGET_OUTPUT;
-        swap_chain_desc.SwapEffect       = DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL;
+        swap_chain_desc.SwapEffect       = DXGI_SWAP_EFFECT_FLIP_DISCARD;
```

Comment:
DXGI_SWAP_EFFECT_FLIP_DISCARD is supposedly slightly faster. I couldnt find any reason why we were using DXGI_SWAP_EFFECT_FLIP_SEQUENTIAL